letsdrill.rb

```diff
@@ -1,13 +1,18 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer >= 90 
+    "A"
+  elsif integer >= 80 
+    "B"
+  elsif integer >= 70 
+    "C"
+  elsif integer >= 60 
+    "D"
+  else "F"
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  array.sort {|a, b| a.length <=> b.length}.shift
```

Comment:
check out `sort_by`.  And then check out `min_by`.  [here](http://ruby-doc.org/core-2.2.3/Enumerable.html)